environments/argocd-app-templates/appsetup-stable.yaml

```diff
@@ -0,0 +1,38 @@
+###############################################################
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
```

Comment:
please change to current year, as the file was just created

Reply:
the year was updated